package.json

```diff
@@ -9,12 +9,12 @@
         "build": "npm-run-all pre-build clear-dist compile",
         "build-artifacts": "npm-run-all build make-dist",
         "bundle": "cd fortigate-autoscale && npm unlink --no-save @fortinet/autoscale-core && cd .. && npm pack ./fortigate-autoscale",
-        "clear-dist": "rimraf */dist",
-        "compile": "tsc --build",
+        "clear-dist": "npx rimraf dist/ && npx rimraf fortigate-autoscale/dist/",
```

Comment:
why do you need npx? the bin directory should already be in the path when running scripts

Reply:
done